src/game/lara1gun.c

```diff
@@ -164,27 +164,22 @@ void AnimateShotgun()
             }
         }
     } else {
-        if (ani == AF_SG_AIM && Input.action) {
-            ani++;
-        } else if (ani > AF_SG_AIM && ani < AF_SG_DRAW) {
-            ani++;
-            if (ani == AF_SG_DRAW) {
-                ani = Input.action ? AF_SG_RECOIL : AF_SG_UNAIM;
-            }
-        } else {
-            if (ani == AF_SG_RECOIL) {
+        if (T1MConfig.enable_shotgun_target_fix && Input.action
+            && !Lara.target) {
+            if (ani >= AF_SG_AIM && ani < AF_SG_DRAW) {
+                ani++;
+                if (ani == AF_SG_DRAW) {
+                    ani = AF_SG_RECOIL;
```

Comment:
Case where `Input.Action` is checked for true and false. Here, the false case seems unhandled.